packages/create-starter/src/index.ts

```diff
@@ -56,6 +57,8 @@ export async function main() {
     process.exit(1);
   }
 
+  trackSelectedKit(options.kit)
```

Comment:
You should `await` this. If the CLI exits before the async call, it will abort and we lose the data.

Reply:
done